app/Models/Course.php

```diff
@@ -27,4 +31,17 @@ public function words()
     {
         return $this->hasMany(Word::class, 'course_id', 'id');
     }
+
+    public function getIsEnrolledAttribute()
+    {
+        $user = Auth::user();
+        $courseId = $this->id;
+        $result = User::where('id', $user->id)
+            ->whereHas('courses', function($q) use ($courseId){
```

Comment:
wherehas là như nào e

Reply:
Dạ anh. Vì Bảng User và bảng Course quan hệ nhiều nhiều với nhau. Em muốn kiểm tra xem có record ở trong bảng trung gian hay không , Where has ở đây giống như 'courses' inner join 'course_user' where user.id = .. and course.id =..